tests/installation/await_install.pm

```diff
@@ -173,7 +169,7 @@ sub run {
             send_key 'alt-o';    # ok
             next;
         }
-        if (get_var('LIVECD') and (match_has_tag('screenlock') || match_has_tag('blackscreen'))) {
+        if (get_var('LIVECD') and (check_screen('screenlock') || check_screen('blackscreen'))) {
```

Comment:
Just wondering how is it better of multi tag assert screen we did previously? Does it help?

Reply:
On this line it does not actually matters. But it matters on the previous steps:

Previously 'screenlock' and 'blackscreen' were part of @tags array. Then this array was used in while loop on line 132: `my $ret = check_screen \@tags, 30;`. This check was designed to define if installation is finished ('rebootnow' needle) or interrupted with 'yast_error', 'yast2_wrong_digest' or other popups. For some reason 'screenlock' and just recently 'blackscreen' were added to this @tags array. Maybe it made sense in the past, but now this leads to the incorrect detecting of the installation state.

I removed these tags from @tags array. And on this line I just changed to `check_screen`, because obviously it will not work with `match_has_tag`.

Reply:
From the code I don't see any difference. As we have this condition in the end of the list. So to me those look as equivalent (except timing), so if you say that helps, we can keep it.

Reply:
This seems like a broken change to me. In the `screenlock` or `blackscreen` case this code isn't reached due to the `next;` in line 143, is it?